### DISCUSSION TITLE
auto-populate a field in a form with the nocodb user's username [Apr 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1997] [Discord Threads]
> 🙋‍♂️ **David (he/him)**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### I think not. can you show an detailed example?  
#  
> 🙋‍♂️ **David (he/him)**  
#### I'm thinking about like a table of volunteers where volunteer.username would be the username they use to log into nocodb. Then a table of work shifts that link to that volunteer table. Then a form view for the work shifts table such that a volunteer could view this form and select a work shift to volunteer for, but they wouldn't have to select the volunteer field in the form because they are already logged in with the username associated with that volunteer row in the volunteer table. So it would be auto-populated, you know?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969293079561502761)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
